common/djangoapps/util/milestones_helpers.py

```diff
@@ -175,6 +175,19 @@ def fulfill_course_milestone(course_key, user):
         milestones_api.add_user_milestone({'id': user.id}, milestone)
 
 
+def remove_course_milestone(course_key, user):
+    """
+    Marks the course specified by the given course_key as in-complete for the given user.
```

Comment:
I'd still prefer to rename this method to reflect the iterative nature of the workflow